docs/fuel-101/relationship-to-ethereum.mdx

```diff
@@ -0,0 +1,76 @@
+
+### State Minimized Techniques
+
+Fuel introduces several state-minimized techniques that leverage bandwidth and execution over state storage, presenting developers with new ways for building other than just smart contract storage:
+
+- Scripts: Ephemeral logic is included in transactions, not stored in state. Unlike EVM transactions which can call a contract directly (but can only call a single contract), Fuel transactions execute a script, which may call zero or more contracts.
+
+- Predicates: Lightweight, stateless programs. A predicate is a new, pure mechanism for authorizing transactions. A predicate can only access the data in a transaction, it cannot view the current chain state. Predicates can be used, among other things, to enable native (stateless) account abstraction.
+
+### A New State-Minimized Transaction Model
+
+By combining these state-minimized techniques with the UTXO model, Fuel offers a flexible transaction model that supports complex, multi-party transactions without the need for traditional smart contracts to access state. This approach also enhances light client verifiability, ensuring the security and integrity of transactions with minimal state footprint.
+
+For a deeper analysis of how Fuel solves for Ethereum state bloat, read [Mastering the Final Boss in Blockchain Scalability: State Growth](https://fuel.mirror.xyz/OrgKx3EOE07YfPydGbQh1_R01lUqH0U0-uyrrEjpgrw) in the Fuel blog.
+
+## Interoperability with Ethereum
+
+Fuel's commitment to interoperability with Ethereum is a cornerstone of its design, facilitating seamless integration between Fuel and the broader Ethereum ecosystem. This interoperability is not just a feature but a fundamental part of Fuel's architecture, enabling it to serve in a way that complements Ethereum's existing infrastructure.
+
+### Modular Architecture
+
+Fuel is designed with modularity at its core, allowing for easy interaction with various data availability and settlement layers. This flexibility ensures that Fuel can adapt to a range of scenarios, from operating alongside Ethereum as a Layer 2 solution to functioning independently or in conjunction with other systems. The modular nature of Fuel's architecture affirms its ability to provide high-performance execution without sacrificing the security and decentralized principles of the Ethereum network.
+
+### Trust-Minimized Bridge
+
+The [Fuel native bridge](https://app.fuel.network/bridge) moves assets currently between Ethereum's Sepolia network and Fuel. It allows for seamless interoperability between Ethereum and Fuel. This bridging capability is essential for maintaining liquidity across the two ecosystems and ensures that users and developers can move assets freely.
```

Comment:
```suggestion
The [Fuel native bridge](https://app.fuel.network/bridge) moves assets currently between Ethereum and Fuel. This bridging capability is essential for maintaining liquidity across the two ecosystems and ensures that users and developers can move assets freely.
```

Reply:
Testnets are always changing we can leave this part out as it will serve mainnet Ethereum in a couple months anyways.